File: FASE4.js
var leapYear = false;
var if_true = "I was born in a leap year.";
var if_false = "I was not born in a leap year.";

var name = "Oriol";
var firstSurname = "Muniz";
var secondSurname = "Baguena";

var day = 2;
var month = 4;
var year = 1997;

// Check if given year is a Gregorian calendar leap year
if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0) {
    leapYear = true;
}

console.log("My name is " + name + " " + firstSurname + " " + secondSurname);
console.log("I was born on " + day + " / " + month + " / " + year);

if (leapYear) {
    console.log(if_true);
}
else {
    console.log(if_false);
}

File: FASE4.cs
using System;

namespace FASE_4
{
    class Program
    {
        static void Main(string[] args)
        {
            bool leapYear = false;
            string yes = "I was born in a leap year";
            string no = "I was not born in a leap year";

            string name, firstSurname, secondSurname;
            name = "Oriol";
            firstSurname = "Muñiz";
            secondSurname = "Baguena";

            int day, month, year;
            day = 2;
            month = 4;
            year = 1997;

            Console.WriteLine($"My name is {name} {firstSurname} {secondSurname}");
            Console.WriteLine($"I was born on {day} / {month} / {year}");

            // Check if bith year was a leap year (Gregorian calendar) 
            if ((year % 4 == 0 && year % 100 != 0) || year % 400 == 0)
            {
                leapYear = true;
            }

            if (leapYear == true)
            {
                Console.WriteLine(yes);
            }
            else
            {
                Console.WriteLine(no);
            }
        }
    }
}
